airgun/views/all_hosts.py

```diff
@@ -605,3 +607,61 @@ class DisassociateHostsModal(PF5Modal):
     @property
     def is_displayed(self):
         return self.browser.wait_for_element(self.title, exception=False) is not None
+
+class ChangeOrganizationLocationModal(PF5Modal):
+    """
+        This class represents 'Change organization/location' modal that is used to change org and location
+        for one or more hosts.
+    """
+    OUIA_ID = 'bulk-assign-taxonomy-modal'
+
+    # Organization section
+    organization_select = PF5Select(locator='.//div[@data-ouia-component-id="select-organization"]')
```

Comment:
do, at the top of the file:
`from widgetastic_patternfly5.ouia import Select as PF5OUIASelect`
and use it like this
```suggestion
    organization_select = PF5Select('select-organization')
```